src/ShellClients/HideTracker.vala

```diff
@@ -21,14 +21,17 @@ public class Gala.HideTracker : Object {
     private bool focus_overlap = false;
     private bool focus_maximized_overlap = false;
 
+    private Meta.Window current_focus_window;
+
     private uint update_timeout_id = 0;
 
     public HideTracker (Meta.Display display, PanelWindow panel) {
         Object (display: display, panel: panel);
     }
 
     construct {
-        var current_focus_window = display.focus_window;
+        // Can't be local otherwise we get a reference cycle :(
```

Comment:
Can you add a link to the vala issue report here?

Reply:
Done :)